### DISCUSSION TITLE
Mining helmet

### DISCUSSION THREAD

**Discussion by ghost** [TechReborn/TechReborn#2514] [Ideas]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
Iron helmet + battery + LED lamp makes a mining helmet.
This helmet provides regional lighting around the player, at the same level a torch would do, in return for energy use.  If possible the helmet would also have an armor value and durability, just like an iron helmet.

**Describe alternatives you've considered**
N/A

**Additional context**
Pretty much the only reason I ever use optifine is the ability to hold a torch in the off-hand to light the immediate area.  Having a mining helmet to add this feature would be great.

**Comment by WrongWeekend** [comment]
I think night vision goggles may be easier to implement.

**Comment by JessiAuro** [comment]
> I think night vision goggles may be easier to implement.

_Far_ easier. The quantum armour currently gives you water breathing in much the same manner, by giving you an infinite water breathing potion while under water.

Doing real dynamic lighting in minecraft though? ooooooooooooof

There's a reason it takes a big-ass mod like optifine to pull it off.

There would be the issue of implementing the means of toggling it however. From what I've seen, TR has no hotkey or the underlying code to support a toggle key, just right-clicking with an active item.
